index.html

```diff
@@ -0,0 +1,201 @@
+  - **should** pass through unknown directives, types, and other schema elements
+  - **should** ignore meaningless URL components (such as unknown query parameters or fragments) within `@using` URLs
+
+An exception to this is processors which prepare the schema for final public consumption. Such processors **may** choose to eliminate all unknown directives and prefixed types in order to hide schema implementation details within the published schema. This will impair the operation of tooling which relies on these directives—such tools will not be able to run on the output schema, so the benefits and costs of this kind of information hiding should be weighed carefully on a case-by-case basis.
+
+## Spec URLs
+
+```graphql
+#::[Declaration of using_URL scalar]
+scalar using_URL
+  @specifiedBy(url: "https://apollographql.com/docs/specs/using#spec-urls")
+```
+
+The URL of a specification.
+
+Specification URLs **must** be valid [RFC 3986 URLs](https://tools.ietf.org/html/rfc3986). They **should** provide the content of the appropriate version of the specification in some human-readable form. It is not required that the content be machine-readable in any particular way.
+
+The final component of the URL's `"/"`-separated `path` **must** be a valid [SemVer specifier](#semver-specifiers) identifying the version of the spec in use. Everything prior to the version specifier is the spec's *base URL*. Processors **should** treat the spec's base URL as an opaque string identifying the specification.
+
+This spec assigns no meaning to additional URL components, such as the query string or fragment. Processors **should** ignore any meaningless URL components when parsing the base URL and semver specifier.
+
+```graphql
+#::[`@using` directive with a query string]
+schema
+  @using(spec: "https://spec.example.com/exampleSpec/1?unrelated=value")
+  { ... }
+```
+
+This directive:
+- Declares that the document conforms to a spec which can be found at `https://spec.example.com/exampleSpec/1`
+- Has a base URL of `https://spec.example.com/exampleSpec`
+- Has a [semver specifier](#semver-specifier) of `1`
+- Has a query string of `"unrelated=value":
+  - This has no effect unless `exampleSpec` assigns it specific meaning.
+
+## Semantic Versioning
+
+Specifications **should** provide a semantic version number.
+
+### SemVer Specifiers
+
+The final `/`-separated path component **must** be a semver specifier. Semver specifiers take the form `v*major*.*minor*.*patch*-*prerelease*`. The minor, patch, and pre-release specifiers are all optional. Examples:
+
+```
+#::[Valid SemVer specifiers]
+v1
+v1.2
+v1.2.3-alpha
+v2.2.3-alpha.2
+```
+
+### SemVer selection
+
+Processors **should** select the highest available spec implementation which is compatible with the requested version according to the [SemVer 2.0.0](https://semver.org/spec/v2.0.0.html) rules for version compatibility. For example, consider this directive:
```

Comment:
Does the linked spec assign any useful semantics to `v1` or `v1.2`? (We could make this more clear by explicitly defining `v1` to be equivalent to `v1.0.0`. Or just by requiring the basename to be a full semver and disallowing `v1`.)

Are we including semver's special cases around v0?

Reply:
I don't think this has been addressed.

Reply:
correct